osu.Game/Tests/Visual/SkinnableTestScene.cs

```diff
@@ -52,7 +52,9 @@ private void load(AudioManager audio, SkinManager skinManager)
 
         private readonly List<Drawable> createdDrawables = new List<Drawable>();
 
-        public void SetContents(Func<Drawable> creationFunction)
+        public void SetContents(Func<Drawable> creationFunction) => SetContents(_ => creationFunction?.Invoke());
```

Comment:
This is the most confusing line I've read in a long time...

Reply:
I've just replaced this override by updating all existing usages to accept the new argument. I think this feels better.

Reply:
Seems good, I avoided it because usually the content shouldn't be dependent on the cell's `ISkin`, but it doesn't really matter so either way is fine.